jobs/resend_invitation_email/worker.go

```diff
@@ -90,15 +90,15 @@ func (rseworker *ResendInvitationEmailWorker) DoJob_24(job *model.Job) {
 
 func (rseworker *ResendInvitationEmailWorker) DoJob_24_48(job *model.Job) {
 	elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_48, _ := rseworker.GetDurations(job)
-	rseworker.Execute(job, elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_48)
+	rseworker.Execute(job, elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_48, "24", "48")
 }
 
 func (rseworker *ResendInvitationEmailWorker) DoJob_24_72(job *model.Job) {
 	elapsedTimeSinceSchedule, DurationInMillis_24, _, DurationInMillis_72 := rseworker.GetDurations(job)
-	rseworker.Execute(job, elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_72)
+	rseworker.Execute(job, elapsedTimeSinceSchedule, DurationInMillis_24, DurationInMillis_72, "24", "72")
 }
 
-func (rseworker *ResendInvitationEmailWorker) Execute(job *model.Job, elapsedTimeSinceSchedule, firstDuration, secondDuration int64) {
+func (rseworker *ResendInvitationEmailWorker) Execute(job *model.Job, elapsedTimeSinceSchedule, firstDuration, secondDuration int64, firstInterval, secondInterval string) {
```

Comment:
Having `firstDuration` and `firstInterval` is a bit confusing and not readable here. Do we need these two and if we do, could there be something we can do to differentiate them with a more meaningful name?

Reply:
Yes we need both. `firstDuration` and `secondDuration` just represent the durations after which the reminders should be sent. For example, when we have flag set to `48` , when the job runs, it'll first send invitation after firstDuration, in this case we decided it would be 24 hours and then it'll run again after  secondDuration which in this case is 48 hours before stopping the job for this particular invite.

For `firstInterval` and `secondInterval` args, they're just for telemetry. For example, knowing that we resend invitations after 24 hours and again after 48 hours their values would `firstInterval="24"` and `secondInterval=48`. We use these values in the email template invite link to know which of the reminders the user used to join. 24? or 48? 

For readability, I can find a more differentiating name.

Reply:
Agreed.